sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/query.sql

```diff
@@ -0,0 +1,52 @@
+-- Aggregated users data joined with Country Lookup
+SELECT
+  submission_date,
+  country.name AS country,
+  city,
+  activity_segment AS segment,
+  normalized_app_name AS app_name,
+  app_version AS app_version,
+  normalized_channel AS channel,
+  distribution_id AS distribution,
+  normalized_os AS os,
+  normalized_os_version AS os_version,
+  os_version_major,
+  os_version_minor,
+  locale AS language,
+  attribution_content,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_experiment,
+  attribution_variation,
+  COUNT(DISTINCT client_id) AS mau,
```

Comment:
For metrics, could we add:
- new_profiles
- ad_click - aggregation is sum
- organic_search_count - sum aggregation
- search_count - sum aggregation
- search_with_ads - sum aggregation

Reply:
Yes, It makes sense to have them here already, no need to have a separate AGG. I added them, we need to double check numbers!

Reply:
The search_counts are fixed in [PR 2956](https://github.com/mozilla/bigquery-etl/pull/2956)